src/freenet/l10n/freenet.l10n.de.properties

```diff
@@ -1037,7 +1037,7 @@ N2NTMToadlet.sendMessageShort=Nachricht senden
 N2NTMToadlet.sendStatus=N2NTM-Sende-Status
 N2NTMToadlet.sent=Nachricht wurde an den Partner gesendet
 N2NTMToadlet.sentTitle=Gesendet
-N2NTMToadlet.tooLong=N2NTMs sind auf 1024 Zeichen beschränkt
+N2NTMToadlet.tooLong=N2NTMs sind auf 131.072 Zeichen beschränkt
```

Comment:
I wouldn’t change this text for languages that you don’t know (putting this on the German text because I know that you actually do know it) because other languages may have different rules for numbers ending in -2 as opposed to numbers ending in -4. :)

Reply:
uh, yes … I was sloppy here.